api/v1beta1/manila_types.go

```diff
@@ -148,6 +152,31 @@ func init() {
 	SchemeBuilder.Register(&Manila{}, &ManilaList{})
 }
 
+// DBPurge struct is used to model the parameters exposed to the Manila API CronJob
+type DBPurge struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=30
+	// Age is the DBPurgeAge parameter and indicates the number of days of purging DB records
+	Age int `json:"age"`
```

Comment:
Will you add the "+kubebuilder:validation:Minimum=1" in this PR (after CI issues are resolved) or in a follow-up patch?

Reply:
I'm going to add this line in a follow up patch because I have that one line change that allows me to focus more on the CI (zuul based) execution rather than the change introduced by the patch.